sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_qa/_qa.py

```diff
@@ -55,6 +55,7 @@ class QAEvaluator:
         }
     """
 
+    # TODO re-enable parallelism once a fix is in place
```

Comment:
Nit: can we create a work item and link the number to this comment to track

Reply:
Oh, I already did, I just forgot to link it here because I realized that this eval isn't actually affected by the bug. It's  in my other pr. I'll remove this todo line.